### PR TITLE
Feature/single activity ui

### PR DESCRIPTION
**Pull request by qgolsteyn**
This PR updates single activity to make use of the new FROG UI components. It groups all data manipulation logic for SingleActivity inside a single folder to reduce the amount of work done inside React components. It also updates Storybook stories to improve navigation (UI components under the UI "folder", App components inside the App "folder).

Feedback, especially from @AbhinavThukral97 would be appreciated. The UI has room to improve, but this work should be done once we agree on the FROG branding and after having created other critical UI components (especially typography related components)

### CODE REVIEW

File: frog/package.json [hunk 1]
```diff
@@ -56,8 +56,8 @@
     "mathjs": "^5.4.2",
     "meteor-node-stubs": "^0.4.1",
     "minio": "^7.0.5",
-    "mobx": "5.9.0",
-    "mobx-react": "^5.2.3",
+    "mobx": "5.13.0",
```
Comment (houshuang) [inline frog/package.json:59]: Was this needed to get it to work?

**Review by houshuang — DISMISSED** [review]

File: frog/package.json (hunk 1 above)
Comment (qgolsteyn) [reply frog/package.json:59]: Yup, was needed to support hooks

**Comment by qgolsteyn** [comment]
@houshuang Ready for review:
- Large images no longer break the UI
- Set a max width on the UI
- Fixed some errors React was complaining about

**Review by houshuang — APPROVED** [review]
Great work!

**Pull request merged**